test/integration/forms/form.js

```diff
@@ -1,21 +1,30 @@
 context('Patient Form', function() {
   specify('routing to form', function() {
+    cy
+      .visit('/');
+
     cy
       .visit('/form/1');
 
     cy
       .get('[data-nav-region]')
       .should('not.be.visible');
 
-    // TODO: replace with on-screen navigation when available
     cy
-      .get('[data-nav-region]')
-      .find('[data-patients-region] .app-nav__link')
-      .last()
-      .click({ force: true });
+      .wait(2000);
```

Comment:
waiting doesn't feel great, but not sure the best way to make sure the form is loaded? Should it be stubbed out with form data? Or somesuch?

Reply:
I don't think we should test the contents of the iframe, just the src for now.  _but_ https://github.com/RoundingWell/RWell/blob/develop/cypress/support/commands.js#L42 when we do.

Reply:
I don't think we need the wait to check the `src`?